disnake/interactions/base.py

```diff
@@ -669,16 +689,14 @@ async def defer(self, *, ephemeral: bool = False, with_message: bool = False) ->
 
         Parameters
```

Comment:
Default values should be noted here, at least for `ephemeral`

Reply:
The default for ephemeral has the same behavior as other methods. Not sure why it needs to be documented here.